Jumpscale/data/nacl/NACL.py

```diff
@@ -265,24 +255,26 @@ def _keys_generate(self, words=None):
         will load in this class
         """
         if words:
-            key2 = j.data.encryption.mnemonic.to_entropy(words)
+            # generate key from mnemonic words
+            entropy = j.data.encryption.mnemonic.to_entropy(words)
+            key = nacl.signing.SigningKey(entropy)
```

Comment:
Should `.encode()` not be called on the SignkingKey here to get the key bytes, such at is done below when a new key is generated?

Reply:
indeed, thx